spec.html

```diff
@@ -36174,7 +36300,8 @@ <h1>
             1. If _fullUnicode_ is *true*, set _e_ to GetStringIndex(_S_, _e_).
             1. If _global_ is *true* or _sticky_ is *true*, then
               1. Perform ? Set(_R_, *"lastIndex"*, 𝔽(_e_), *true*).
-            1. Let _n_ be the number of elements in _r_'s _captures_ List. (This is the same value as <emu-xref href="#sec-notation"></emu-xref>'s _NcapturingParens_.)
+            1. Let _n_ be the number of elements in _r_'s _captures_ List.
+            1. Assert: _n_ = _R_.[[RegExpRecord]].[[NcapturingParens]].
```

Comment:
Would this be clearer without the assertion?
```suggestion
            1. Let _n_ be _R_.[[RegExpRecord]].[[NcapturingParens]].
```

Reply:
I like the current way with the assert, since we're using _n_ later in the algorithm to iterate through _captures_.